### PR TITLE
(improvement) TickerList: added filtering by fullnames of ccy

### PR DESCRIPTION
**Pull request by dmytroshch**
## Prerequisites
https://github.com/bitfinexcom/bfx-hf-ui/pull/458

## Task reference
https://app.asana.com/0/1125859137800433/1200403923487576/f

## BREAKING CHANGES
N/A

## PR description
Added filtering by fullnames of ccy. Added optional key of prop "data" - "label"(array of strings)

## Example app screenshot
Снимок экрана от 2021-06-03 20-31-04

## Storybook screenshot
Снимок экрана от 2021-06-03 20-30-54

## Checklist
  - [x] PR title has category name prefix, e.g. "\(fix\) {description}" (fix/feature/refactor/improvement/doc/test)
  - [x] Added PR description
  - [x] Relevant change in example app is verified, added example app screenshot
  - [x] Storybook: stories, docs are updated/verified
  - [x] `Pull request verify workflow` passed
  - [x] PR development is completed, the developer is satisfied with the code quality and behavior of the app

### CODE REVIEW

File: packages/core/src/components/TickerList/TickerList.js [hunk 1]
```diff
@@ -38,6 +38,7 @@ export const TickerList = (props) => {
   const keyForId = getMappedKey(KEYS.ID, rowMapping)
   const keyForBaseCcy = getMappedKey(KEYS.BASE_CCY, rowMapping)
   const keyForQuoteCcy = getMappedKey(KEYS.QUOTE_CCY, rowMapping)
+  const keyForLabels = getMappedKey(KEYS.LABELS, rowMapping)
```
Comment (bhoomij) [inline packages/core/src/components/TickerList/TickerList.js:41]: - rename labels this to `baseCcyFullName`
- change matches code to
```
  const matches = _toLower(_join([
        baseCcy,
        quoteCcy,
        baseCcy + quoteCcy,
        baseCcyFullName,
      ]))
```
- also add this new support to TickerList.container, by passing baseCcyFullName in `data` prop,
- after above changes, I think we don't need backward compatibility line

**Review by bhoomij — CHANGES_REQUESTED** [review]

**Comment by dmytroshch** [comment]
Снимок экрана от 2021-06-04 12-33-41

**Comment by bhoomij** [comment]
for comment
`also add this new support to TickerList.container, by passing ccyLables in data prop`
please create task in modular-ui project, so can handle it in separate PR

**Review by bhoomij — APPROVED** [review]

**Comment by dmytroshch** [comment]
> for comment
> `also add this new support to TickerList.container, by passing ccyLables in data prop`
> please create task in modular-ui project, so can handle it in separate PR @dmytroshch
@bhoomij we don't have TickerList.container
Снимок экрана от 2021-06-04 12-58-12

**Comment by bhoomij** [comment]
> > for comment
> > `also add this new support to TickerList.container, by passing ccyLables in data prop`
> > please create task in modular-ui project, so can handle it in separate PR @dmytroshch
> 
> @bhoomij we don't have TickerList.container
> Снимок экрана от 2021-06-04 12-58-12

@dmytroshch you can find it https://github.com/bitfinexcom/ufx-ui/tree/staging/packages/bfx-containers/src/containers/TickerList
more context:
ufx-ui has two type of components:
packages/core: holds ui components
packages/bfx-containers: uses core comps and injects bfx-data

**Pull request merged**